jest.config.cjs

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  resolver: "jest-ts-webcompat-resolver",
```

Comment:
So glad this stuff exists. Didn't know about this before!
I switched to mocha in webnative mostly because of that issue. I'm not sure this would've saved us there, since we were also using the jest-puppeteer preset, and that overrides some imports.
If we ever end up having issues with mocha in webnative, we can consider jest with this resolver as an alternative I think.